dreadnode/api/client.py

```diff
@@ -736,35 +736,17 @@ def get_container_registry_credentials(self) -> ContainerRegistryCredentials:
         response = self.request("POST", "/platform/registry-token")
         return ContainerRegistryCredentials(**response.json())
 
-    def get_platform_releases(
-        self, tag: str, services: list[str], cli_version: str | None
-    ) -> RegistryImageDetails:
+    def get_platform_releases(self, tag: str, services: list[str]) -> RegistryImageDetails:
         """
         Resolves the platform releases for the current project.
 
         Returns:
             The resolved platform releases as a ResolveReleasesResponse object.
         """
-        payload = {
-            "tag": tag,
-            "services": services,
-            "cli_version": cli_version,
-        }
-        try:
-            response = self.request("POST", "/platform/get-releases", json_data=payload)
-
-        except RuntimeError as e:
-            if "403" in str(e):
-                raise RuntimeError("You do not have access to platform releases.") from e
-
-            if "404" in str(e):
-                if "Image not found" in str(e):
-                    raise RuntimeError("Image not found") from e
+        from dreadnode.version import VERSION
 
-                raise RuntimeError(
-                    f"Failed to get platform releases: {e}. The feature is likely disabled on this server"
-                ) from e
-            raise
```

Comment:
I only removed this as I think it's a poor pattern to reflect in the API client code. Might be aggressive to remove clarity from the errors for users - open to moving this somewhere else in the CLI